### PR TITLE
Add moler ASCII art logo and version into logs

### PR DESCRIPTION
**Pull request by Ernold11**

### CODE REVIEW

File: moler/config/loggers.py [hunk 1]
```diff
@@ -243,6 +284,10 @@ def configure_moler_main_logger():
                                   formatter=MultilineWithDirectionFormatter(fmt=debug_log_format,
                                                                             datefmt=date_format))
 
+        global moler_logo
```
Comment (greg-latuszek) [inline moler/config/loggers.py]: Why you add global declaration here? You don't modify `moler_logo` from inside of this function - you just read it. So, global line is unnecessary.

File: moler/config/loggers.py [hunk 2]
```diff
@@ -27,6 +28,46 @@
...
+    %%  %%%%%  %%  %%%       %%  %%%       %%%        %%%   %%%
+    %%   %%%   %%   %%%%%%%%%%%  %%%%%%%%% %%%%%%%%%% %%%     %%%
+"""
+
+
+def _get_moler_version():
+    try:
+        return pkg_resources.get_distribution("moler").version
+    except pkg_resources.DistributionNotFound:
+        return "cloned from git repository"
```
Comment (greg-latuszek) [inline moler/config/loggers.py]: if it is from git repo you can also retrieve version:
```python
import os
setup_py_path = os.path.join(os.path.dirname(__file__), "..", "..", "setup.py")
```
try if file exists, open it and find version

Comment (Ernold11) [reply moler/config/loggers.py]: something like that?

**Comment by Ernold11** [comment]
Please @greg-latuszek look into code, all comments resolved.

**Comment by greg-latuszek** [comment]
so, ship it :-)

**Pull request merged**